Sources/Journify/Plugins.swift

```diff
@@ -161,3 +161,25 @@ extension Journify {
         return timeline.find(key: key)
     }
 }
+
+// MARK: - Adding Traits Plugin
+
+class InjectTraitsPlugin: Plugin {
+    var analytics: Journify?
+    var type: PluginType = .enrichment
+    
+    public func execute<T: RawEvent>(event: T?) -> T? {
+        var event = event
+        guard event?.type != "identify", let traits = analytics?.traits() as? [String: Any] else {
+            return event
+        }
+        var updatedContext = event?.context
```

Comment:
nitpick: We need to add the `traits` to the event. not event.context

Reply:
@amineck check again, please